invenio_users_resources/resources/users/config.py

```diff
@@ -35,12 +39,13 @@ class UsersResourceConfig(RecordResourceConfig):
     url_prefix = "/users"
     routes = {
         "list": "",
-        "moderation_search": "/moderation",
+        "search_all": "/all",
```

Comment:
Comment: Changing name here, because the API is more than just moderation, it's provides all the admin interface with all the user information